src/users.spec.ts

```diff
@@ -246,24 +247,26 @@ describe('Users...', () => {
         // @note This should be done using the cli
         // This time they create a user key
         const key = await createKey(ctx, 'USER')
-        await ctx.withAPIKey(key.key).withUserKey(key)
-        expect(ctx.toJSON()).to.have.ownProperty('x-textile-api-sig')
+        // Update user context
+        await userContext.withAPIKey(key.key).withUserKey(key)
+        expect(userContext.toJSON()).to.have.ownProperty('x-textile-api-sig')
       }).timeout(3000)
       it('should then generate a user identity and get a token for it', async () => {
         const identity = await Libp2pCryptoIdentity.fromRandom()
-        const db = new Client(ctx)
+        const db = new Client(userContext)
         await db.getToken(identity)
       })
       it('should then create a db for the bucket', async () => {
         const id = ThreadID.fromRandom()
-        const db = new Client(ctx)
-        await db.newDB(id, ctx.withThreadName('my-buckets'))
-        expect(ctx.toJSON()).to.have.ownProperty('x-textile-thread-name')
+        // Just for the test, recrete the client here
+        const db = new Client(userContext)
```

Comment:
Also doing this to demonstrate that, in an app that refreshes on a page load, this would still work if you (say) stored the context in redux or similar.